src/object-diagram/utilities/characterReplacer.ts

```diff
@@ -0,0 +1,37 @@
+type Transformation = (character: string) => string;
+type Range = [number, number, Transformation];
```

Comment:
Why an array and not an object?
`{ start: number; end: number; transformation: Transformation }`

Reply:
You are right, using an object improves readabliliy

Reply:
Done